common/rpc/interceptor/telemetry.go

```diff
@@ -255,14 +257,6 @@ func (ti *TelemetryInterceptor) emitActionMetric(
 	}
 
 	switch methodName {
-	case startWorkflowExecution:
-		resp, ok := result.(*workflowservice.StartWorkflowExecutionResponse)
-		if !ok {
-			return
-		}
-		if resp.Started {
-			metrics.ActionCounter.With(metricsHandler).Record(1, metrics.ActionType("grpc_"+methodName))
-		}
```

Comment:
Is this intentional change?

Reply:
yes, this aligns with the documentation changes that I shared yesterday. we charge startworkflowexecution in cloud even if wf is already started